app/components/InfoHeader.tsx

```diff
@@ -43,6 +48,19 @@ export function InfoHeader({ relatedPaths }: InfoHeaderProps) {
   const [hovering, setHovering] = useState(false);
   console.warn(selectedInfo);
 
+  const newPath = formattedSelectedInfo.replace(/^#/, "$").replace(/\//g, ".");
+
+  const checkPathExists = () => {
```

Comment:
I'd move this function outside of the React component (and use `function checkPathExists` style instead of the function literal